### PR TITLE
Multiple exchanges

### LINKED ISSUE #47: Ability to add more than one account of an exchange
Even thought this might be an issue that won't affect many people, I think it is important to include the possibility for it early on in the development process so that data models etc allow for it.

### PR DESCRIPTION
**Pull request by rakanalh**
Resolves #47 

TODO:
- [ ] Figure out DB upgrades (if any)
- [ ] UI Fixes needed to have support adding multiple exchanges (Need help here)
- [x] Fix tests
- [ ] ?

### CODE REVIEW

**Comment by LefterisJP** [comment]
Hey thanks for this @rakanalh! As discussed I will support you with this as much as needed. But provided v1.0 happens soon this will probably go in after it.

**Comment by rakanalh** [comment]
Sure @LefterisJP. I am keeping as a draf, but wanted to expose my work publicly until v1.0 is released & the UI stuff are updated... so no rush on merging this.

File: rotkehlchen/db/utils.py [hunk 1]
```diff
@@ -19,7 +19,8 @@
 
 DB_CREATE_USER_CREDENTIALS = """
 CREATE TABLE IF NOT EXISTS user_credentials (
-    name VARCHAR[24] NOT NULL PRIMARY KEY,
+    id INTEGER PRIMARY KEY,
```
Comment (rakanalh) [inline rotkehlchen/db/utils.py]: @LefterisJP Do you have an approach for DB upgrades in mind?

Comment (LefterisJP) [reply rotkehlchen/db/utils.py]: Hmm so far it was only needed once. To upgrade the stored ETH accounts to a checksummed format. The commit is this: b31a360dd291ac0539bdf106bfd69817455a3025

The update happens here: https://github.com/rotkehlchenio/rotkehlchen/blob/2193dfe2215f59c97d3bbc809e5ec6edd5afcac5/rotkehlchen/db/dbhandler.py#L226-L238

Essentially if the version is 1 then it does the upgrade and it changes the version to 2. I suppose the same approach can be built upon with version 2->3 e.t.c.

**Comment by LefterisJP** [comment]
Hey @rakanalh I am closing this PR as it's horribly outdated :sweat_smile: 

I would still love to see you contribute to Rotki! It can be a nice python break after all this Rust you are dealing with these days :smile:

**Pull request closed without merging**